DependencyInjection/Compiler/AssetsVersionCompilerPass.php

```diff
@@ -29,6 +29,11 @@
  */
 class AssetsVersionCompilerPass extends AbstractCompilerPass
 {
+    /**
+     * {@inheritdoc}
+     *
+     * @return void
+     */
     public function process(ContainerBuilder $container)
```

Comment:
```suggestion
    public function process(ContainerBuilder $container): void
```
as its not expected that anyone extends these compilar passes, i would prefer to add actual return tye declarations instead of the phpdoc